src/java/picard/sam/FilterSamReads.java

```diff
@@ -20,6 +20,9 @@
  * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
+ * 
+ * History:
+ *  2015-11: Pierre Lindenbaum added the javascript filter
```

Comment:
In general we typically just let git blame keep track of the history.  No need to delete it, but since it's not usually updated it's not usually a reliable source of information.